package.json

```diff
@@ -6,6 +6,7 @@
   "description": "Scripts for the college-choice project.",
   "scripts": {
     "build": "browserify `js/src/browserify-args.sh`",
+    "build:d3": "d3-bundler -- js/src/d3-bundle.js > js/src/d3.js",
```

Comment:
Should d3 be built along with the normal build? Or is it a thing you'd only have to do when you update d3?